### ISSUE TITLE
Import Local Docset

### ISSUE THREAD

**Issue by littleiffel** [jkozera/zevdocs#2]
First big thanks for the project, it looks really promising)

One last step to make it fit my work flow perfectly would be to import a local docset into ZevDocs.

I have a local docset that i don't want to publish so is it possible to integrate them somehow? Thanks

**Comment by jmymay** [comment]
Thanks for the feature request! :) Is your docset in Dash format? If so, then I could quickly provide a CLI tool for importing the docs from a .tgz file. (Later this could be integrated into GUI.)

**Comment by littleiffel** [comment]
@jkozera Yes it can be imported into Dash (tested with another developers Mac)

It would be awesome to be able to do that

**Comment by jmymay** [comment]
@littleiffel Done, I will release `0.5.1` shortly with this implemented. Usage:

1. Download `install_local_docset.sh` from [here](https://raw.githubusercontent.com/jkozera/zealcore/efe1c8e5046aa9d2644cafcfcd708418136dc476/install_local_docset.sh).
2. Open ZevDocs `0.5.1`.
3. Run `install_local_docset.sh path_to_docset.tgz` in your terminal. **It requires nodejs, npm, and python3.** 
4. After you get a result similar to:
```
connected (press CTRL+C to quit)
< {"RepoId":"com.kapeli.local","Docset":"[docset name]","Received":405504,"Total":425726}
< {"RepoId":"com.kapeli.local","Docset":"[docset name]","Received":409600,"Total":425726}
< {"RepoId":"com.kapeli.local","Docset":"[docset name]","Received":413696,"Total":425726}
< {"RepoId":"com.kapeli.local","Docset":"[docset name]","Received":417792,"Total":425726}
< {"RepoId":"com.kapeli.local","Docset":"[docset name]","Received":421888,"Total":425726}
< {"RepoId":"com.kapeli.local","Docset":"[docset name]","Received":425725,"Total":425726}
< {"RepoId":"com.kapeli.local","Docset":"[docset name]","Received":425726,"Total":425726}
disconnected (code: 1000)
Success!
```
then you need to **close ZevDocs**, and open it again.

Hope that helps! :) Not closing the issue yet, because I'd like to implement GUI support too.

**Comment by littleiffel** [comment]
@jkozera Wow Thanks..I am impressed wil try it  as soon as i get the update :)

**Comment by littleiffel** [comment]
Already got the Update
